Quiz.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# consider data bike_sharing in the repo . Respond with your conclusions [and code if needed ] for the following
+# 
+# Are the average number of rented bikes in first 7 days of months different significantly between first 6 months Vs last 6 months of the year ? [consider column cnt]
+# Find the average temperature [consider column temp] in the first quarter of the year . Has this changed significantly in the third quarter of the year ?
+# Does it matter which weekday it is , when it comes to causally rented bikes ? [consider column casual]
+
+# In[1]:
+
+
+import seaborn as sns
+import numpy as np
+import pandas as pd
+
+
+# In[2]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[3]:
+
+
+bike = pd.read_csv('bike_sharing_hours.csv')
+
+
+# In[4]:
+
+
+bike.sample(10)
+
+
+# In[5]:
+
+
+bike.info()
+
+
+# ### Are the average number of rented bikes in first 7 days of months different significantly between first 6 months Vs last 6 months of the year ? [consider column cnt]
+
+# In[6]:
+
+
+bike['dteday'] = pd.to_datetime(bike['dteday'])
+
+
+# In[7]:
+
+
+bike[['cnt','dteday','mnth']]
+
+
+# In[8]:
+
+
+bike['days']=bike.dteday.dt.day
+
+
+# In[9]:
+
+
+from scipy.stats import levene
+
+
+# In[10]:
+
+
+levene(bike.loc[bike['days']<8,'cnt'],bike.loc[bike['mnth']<7,'cnt'])
+
+
+# In[11]:
+
+
+from scipy.stats import ttest_ind
+
+
+# In[14]:
+
+
+ttest_ind(bike.loc[bike['days']<8,'cnt'],
+          bike.loc[bike['mnth']<7,'cnt'],
+          equal_var=False)
+
```

Comment:
questions is about bikes rented in first 7 days . for first 6 months vs last 6 months . that's not what your solution is doing . 

bike.loc[bike['days']<8,'cnt'] = this is bikes rented in first 7 days of all months 
 bike.loc[bike['mnth']<7,'cnt'] = this is bikes rented for first 6 months , all days